### PR TITLE
[test] Replace OT const with OTEL

### PR DESCRIPTION
**Pull request by afzal442**
## Which problem is this PR solving?
* Part of #3381 
* This PR adds `otel trace` spans type to span model

## Short description of the changes
- Replace OT const with `otel trace` spans const i.e. `ext.SpanKind..` with `trace.SpanKind..`

### CODE REVIEW

File: cmd/query/app/handler_options.go [hunk 1]
```diff
@@ -61,10 +61,10 @@ func (handlerOptions) QueryLookbackDuration(queryLookbackDuration time.Duration)
 	}
 }
 
-// Tracer creates a HandlerOption that initializes OpenTracing tracer
+// Tracer creates a HandlerOption that initializes Opentelemetry tracer
 func (handlerOptions) Tracer(tracer *jtracer.JTracer) HandlerOption {
 	return func(apiHandler *APIHandler) {
-		apiHandler.tracer = tracer
+		apiHandler.tracer.OTEL = tracer.OTEL
```
Comment (yurishkuro) [inline cmd/query/app/handler_options.go:67]: why this change?

Comment (afzalbin64) [reply cmd/query/app/handler_options.go:67]: I did this to workaround if it was desirable.
